en/docs/quick-starts/age-based-adaptive-auth.md

```diff
@@ -4,12 +4,6 @@ This page guides you through configuring user age-based adaptive authentication
 
 ----
 
-If you have your own application, click the button below.
-
-<a class="samplebtn_a" href="../../guides/configure-adaptive-auth"   rel="nofollow noopener">I have my own application</a>
```

Comment:
currently, we don't have a guide for this?

Reply:
No